### PR TITLE
1976 line spacing

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/src/presenter/actions/printTrialCalendarAction.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { state } from 'cerebral';
-import printTrialCalendarTemplate from '../../views/TrialSessionDetail/printTrialCalendarTemplate.html';
+import printTrialCalendarTemplate from '../../views/TrialSessionDetail/printTrialCalendarTemplate.htm';
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/printTrialCalendarAction.js:2]: Why this change to `.htm`? You'll also need to update `web-client/jest.config.js` to correctly transform `.htm` files.

File: web-client/src/presenter/actions/printTrialCalendarAction.js [hunk 2]
```diff
@@ -17,18 +17,17 @@ export const printTrialCalendarAction = ({ get }) => {
 
   const renderCases = item => {
     return `<tr>
-      <td>
+      <td class="valign-top">
         ${item.docketNumberWithSuffix}
       </td>
-      <td>${item.caseCaption}</td>
-      <td style="width: 25%;">
+      <td class="valign-top" class="line-height-13">${item.caseCaption}"</td>
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/printTrialCalendarAction.js]: I think there's an extra `"` in here after the caseCaption.

File: web-client/src/presenter/actions/printTrialCalendarAction.js (hunk 1 above)
Comment (sutt0n) [reply web-client/src/presenter/actions/printTrialCalendarAction.js:2]: This was changed to `.htm` to trigger Parcel's HMR on-save.

**Review by rachaelparris — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]
I also welcome future efforts to utilize some kind of well-established templating engine to replace these efforts. :)
